core/dbt/include/global_project/macros/materializations/seed/seed.sql

```diff
@@ -47,6 +42,21 @@
     {{ return(sql) }}
 {% endmacro %}
 
+{% macro get_binding_char() -%}
+  {{ adapter.dispatch('get_binding_char')() }}
+{%- endmacro %}
+
+{% macro default__get_binding_char() %}
+  {{ return('%s') }}
+{% endmacro %}
+
+{% macro get_batch_size() -%}
+  {{ adapter.dispatch('get_batch_size')() }}
+{%- endmacro %}
+
+{% macro default__get_batch_size() %}
+  {{ return(var("batch_size", 10000)) }}
+{% endmacro %}
```

Comment:
Definitely agree with turning this from an embedded hard-coded value into a standalone macro!

I don't think I see the need for the `var` here. As it is, users can override/reimplement `get_batch_size()` in their root projects, and dbt will prefer their version. So I'd suggestion just:
```suggestion
{% macro default__get_batch_size() %}
  {{ return(10000) }}
{% endmacro %}
```

Reply:
I'm trying to decide if this needs to be a _dispatched_ macro. How common would it be that an adapter needs to change the default from 10k because of some implementation detail of their database / python client / pyodbc / etc? (I _think_ I follow the reasoning in https://github.com/dbt-msft/dbt-sqlserver/pull/151, which would seem to be a case in point.)

There's very little downside to dispatching, only more optionality at the cost of (very minor) complexity, so I don't really have any opposition.